### PR TITLE
added method edit_message_media in CallbackQuery

### PR DESCRIPTION
**Pull request by higoramp**
Since the telegram API allows edit media on a inlineMessage, we should be able to do so

### CODE REVIEW

File: telegram/callbackquery.py [hunk 1]
```diff
@@ -196,3 +196,27 @@ def edit_message_reply_markup(self, reply_markup, *args, **kwargs):
                                                       chat_id=self.message.chat_id,
                                                       message_id=self.message.message_id,
                                                       *args, **kwargs)
+
```
Comment (jsmnbom) [inline telegram/callbackquery.py]: flake8 has some issues with this, 
image

File: telegram/version.py [hunk 2]
```diff
@@ -17,4 +17,4 @@
 # You should have received a copy of the GNU Lesser Public License
 # along with this program.  If not, see [http://www.gnu.org/licenses/].
 
-__version__ = '12.0.0b1'
+__version__ = '12.0.1b1'
```
Comment (jsmnbom) [inline telegram/version.py]: Please don't update the version in PRs, it leads to a lot of merge conflicts :)

**Comment by Eldinnie** [comment]
@higoramp Are you still with us on this?

**Comment by Eldinnie** [comment]
closing due to inactivity

**Pull request closed without merging**